sdk/tables/azure-tables/src/utils/bufferSerializer.browser.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export function isUint8Array(value: any): boolean {
+  return value instanceof Uint8Array;
+}
+
+/**
+ * Encodes a byte array in base64 format.
+ * @param value the Uint8Aray to encode
+ */
+export function encodeByteArray(value: Uint8Array): string {
```

Comment:
NIT: I would prefer if "base64" was somehow worked into this function name (and decode string) so it was clear at the calls sites what sort of encoding was going on.  Would `base64Encode` and `base64Decode` work as names for these?